src/templates/components/Pixel.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	export let src: string;
+</script>
+
+<img {src} alt="" class="pixel" />
+
+<style lang="scss">
+	.pixel {
+		position: absolute;
+		bottom: 0;
+		right: 0;
+		width: 1px;
+		height: 1px;
+		opacity: 0;
+	}
+</style>
```

Comment:
As Svelte scopes styles automaticall, you can apply the styles directly to `img` without the need for a specific class name.